1 explicit pages:
  config/english/config_en.prc

2 implicit pages:
  /c/Panda3D-1.8.1/etc/Config.prc
  /c/Panda3D-1.8.1/etc/Confauto.prc

serverVersion:  sv-1.0.0
PAccount: accountServer from launcher:  
PAccount: default accountServer:  https://pirates.go.com
Initializing quest ladders...
:MagicWordManager(warning): You risk being banned if you abuse this mate!
:MagicWordManager(warning): I will have Jack cut you into pieces! (a.k.a. banned forever)
:DistributedObjectGlobal(warning): ChatManager going online
:DistributedShipLoader(warning): ShipLoader going online
Traceback (most recent call last):
  File "C:\Panda3D-1.8.1\python\lib\runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Panda3D-1.8.1\python\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Users\Tyler\Desktop\POTCOR-src-master\POTCOR-src-master\pirates\piratesbase\PiratesStart.py", line 62, in <module>
    cr = PiratesClientRepository.PiratesClientRepository(serverVersion, launcher)
  File "pirates\distributed\PiratesClientRepository.py", line 140, in __init__
    self.tutorialObject = DistributedPiratesTutorial(self)
  File "pirates\tutorial\DistributedPiratesTutorial.py", line 201, in __init__
    base.cr.tutorialObject = self
AttributeError: PiratesBase instance has no attribute 'cr'
